### PR TITLE
Fix regression tests on windows

### PR DESCRIPTION
**Pull request by bakpaul**
And some cleaning removing parameters not needed anymore thank's to last PR adding regression test path inside the test file.

### CODE REVIEW

**Comment by bakpaul** [comment]
The actual fix is in the file unit-tests.sh lines 32-39. The rest is only refacto

**Review by fredroy — APPROVED** [review]
Not really an expert but looks not bad at least.
Just a grin of the "unit-tests.sh" which can run unit and... regression tests 🥸
An other refactoring would be needed 😅

File: scripts/unit-tests.sh [hunk 1]
```diff
@@ -9,25 +9,34 @@
...
-    if [ -n "$4" ]; then
+    if [ "$2" == "regression" ]; then
         test_type="regression-tests"
-        references_dir="$4"
+        if vm-is-windows; then
+            # Avoid "libpython3X not found"
+            if [ -e "$VM_PYTHON3_EXECUTABLE" ]; then
+                pythonroot="$(dirname $VM_PYTHON3_EXECUTABLE)"
+                pythonroot="$(cd "$pythonroot" && pwd)"
+                export PATH="$pythonroot:$pythonroot/DLLs:$pythonroot/Lib:$PATH_RESET"
```
Comment (epernod) [inline scripts/unit-tests.sh:37]:
```suggestion
                export PATH="$pythonroot:$pythonroot/DLLs:$pythonroot/Lib:$PATH"
```
why do you need to store in PATH_RESET?

**Review by epernod — APPROVED** [review]
Same for me, not an expert but I don't see any red light.
I just made a small remark regarding PATH_RESET but I probably don't see in the PR where you compute different paths between PATH_RESET and PATH

File: scripts/unit-tests.sh (hunk 1 above)
Comment (bakpaul) [reply scripts/unit-tests.sh:37]: I reset the PATH at the end of the script. It is here not very useful given the fact that this is the last stage of the ci script, but at one point if in the future we add steps, it is better to clean up after...

**Pull request merged**